Makefile

```diff
@@ -42,12 +44,16 @@ ALL_OS_ARCH = $(foreach os, $(ALL_OS), ${ALL_OS_ARCH.${os}})
 # The current context of image building
 # The architecture of the image
 ARCH ?= amd64
+CONFORMANCE_ARCH ?= linux/amd64,linux/arm64
```

Comment:
We can rely on https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/cdfff9350e34ce1acc2a108738d0eca2d94f9886/Makefile#L36